smu.c

```diff
@@ -983,8 +1011,10 @@ u32 smu_update_pmtable_size(u32 version) {
   case CODENAME_REMBRANDT:
     switch (version) {
     case 0x450004:
+      g_smu.pm_dram_map_size = 0xAA4;
+      break;
     case 0x450005:
-      g_smu.pm_dram_map_size = 0xA44;
+      g_smu.pm_dram_map_size = 0xAB0;
```

Comment:
is this value correct?
this breaks ryzenadj due to table size mismatch

Reply:
AFAICS that's the value used by AMD in Ryzen Master. So it is as correct as it can get. Theoretically we can also cross-check it by digging in UEFI image but I never tried it.

Something breaks if you change hardcoded table size in ryzenadj's lib/api.c to match?